frontend/src/common/enums/ui/button-style.enum.ts

```diff
@@ -0,0 +1,7 @@
+enum ButtonStyle {
+  PRIMARY = 'button',
+  LIGHT = 'buttonLight',
```

Comment:
```suggestion
  PRIMARY = 'primary',
  LIGHT = 'light',
```
I guess it should be
![image](https://user-images.githubusercontent.com/60396171/127331831-8e732685-98d2-4f67-8894-c780efd23446.png)